yajsapi/rest/common.ts

```diff
@@ -28,7 +28,7 @@ export async function repeat_on_error(
   block: Callable<void, any>,
   max_tries: number = 5,
   max_duration_ms = 15000,
-  interval: number = 0.0,
+  interval: number = 1000,
```

Comment:
Maybe rename this to `interval_ms`? Since the original value was `0.0`, I'd guess that `interval` is in seconde, not ms.

Reply:
Fixed